zadaci/sa-kolokvijuma/2020/RA/T5/G9/index.rst

```diff
@@ -0,0 +1,18 @@
+.. index::
+   Algoritmi ; Pretraga
+   Datoteke ; Tekstualne
+   Dinamičke strukture ; Lista
+
+
+Zadatak "Magacin"
+====================
```

Comment:
Give more descriptive name, for example "Magacin mobilne i računarske opreme" and make sure you have as much of = as your title lenght is.